README.md

```diff
@@ -0,0 +1,116 @@
+Verible Lint Action
+===================
+
+Usage
+-----
+
+See [action.yml](action.yml)
+
+This is a GitHub Action used to lint Verilog and SystemVerilog source files
+and comment erroneous lines of code in Pull Requests automatically.
+The GitHub Token input is used to provide
+[reviewdog](https://github.com/reviewdog/reviewdog)
+access to the PR.
+If you don't wish to use the automatic PR review,
+you can omit the ``github_token`` input.
+If you'd like to use a reporter of reviewdog other than ``github-pr-review``,
+you can pass its name in the input ``reviewdog_reporter``.
+
+Here's a basic example to lint all ``*.v`` and ``*.sv`` files:
+```yaml
+name: Verible linter example
+on:
+  pull_request:
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: antmicro/verible-actions@main
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+You can provide optional arguments to specify paths, exclude paths,
+a config file and extra arguments for ``verible-verilog-lint``.
+
+```yaml
+- uses: antmicro/verible-actions@main
+  with:
+    config_file: 'config.rules'
+    paths: |
+      ./rtl
+      ./shared
+    exclude_paths: |
+      ./rtl/some_file
+    extra_args: "--check_syntax=true"
+    github_token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+Automatic review on PRs from external repositories
+--------------------------------------------------
+
+In GitHub Actions, workflows triggered by external repositories
+may only have read access to the main repository
+\[[docs](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token)\].
```

Comment:
can we make "read access to the main repository" the link? So we don't have to have the link in in brackets.

Reply:
Yes, done.